Fix: $input.current.form undefined/null

agrohs opened the pull request:
Ref checks on $input.current.form are missing undefined checks

agrohs committed:
Fix: $input.current.form undefined/null

Ref checks on $input.current.form are missing undefined checks

Dan503 commented on src/time-polyfill/ReactTimeInputPolyfill.js:
```diff
@@ -109,7 +109,7 @@ export default class TimeInputPolyfill extends React.Component {
 	}
 	componentWillUnmount() {
-		if (this.$input.current.form) {
+		if (this.$input.current && this.$input.current.form) {
```
I'm wondering if we can use optional chaining instead 🤔 

I'm not sure if my build/release process will transform it for IE though.

This is probably the safer option I guess...

agrohs replied:
Optional chaining will work, but depends on rollup/babel config so went with this for now as it was safer (especially as Issue had been open since March)

Commit:
Using optional chaining

I have build processes in place so it will compile them into an IE compatible format

Commit:
Making the time input font-family monospace

Modern browsers display time inputs as monospace

Commit:
Version bump to v1.0.7
